internal/controller/apiextensions/claim/configurator.go

```diff
@@ -185,21 +188,36 @@ func filter(in map[string]any, keys ...string) map[string]any {
 	return out
 }
 
-// APIClaimConfigurator configures the supplied claims with fields
-// from the composite. This includes late-initializing spec values
-// and updating status fields in claim.
-type APIClaimConfigurator struct {
-	client client.Client
-}
+func keep(in map[string]any, keys ...string) map[string]any {
```

Comment:
now that I see `keep`, `filter` looks like a bad name for what it does. Something like `withoutKeys` and `onlyKeys` would be clearer.

Reply:
I agree with this.

Reply:
done, please reopen the conversation if does not match the expectations.